lxd/api_cluster.go

```diff
@@ -3305,7 +3309,7 @@ func restoreClusterMember(d *Daemon, r *http.Request) response.Response {
 
 					// Wait for the forceful stop to complete.
 					err = stopOp.Wait()
-					if err != nil {
+					if err != nil && !strings.Contains(err.Error(), "The instance is already stopped") {
```

Comment:
@stgraber this is a hack, but the operation response codes don't align with HTTP response statuses, so I didn't feel like I could just introduce a 409 conflict one. Do you have any preference on how to introduce the ability to return HTTP response codes to operation responses?

Reply:
Hmm, not sure. Operations aren't directly related to REST API calls, so it makes some sense that they're not expected to return HTTP status codes. But having something better than arbitrary string matching would be nice though.